release-engineering/selection.md

```diff
@@ -0,0 +1,85 @@
+# Release Manager selection
+
+> DISCLAIMER: The initial draft of this document is intended to capture the
+> current state of the Release Manager selection process. It is neither a
+> perfect, nor complete process, but it is one that we have at present,
+> so hopefully documenting it is a step in the right direction.
+
+As described on the Kubernetes Community [website][release-managers-page], the
+Release Managers group is a group of Kubernetes contributors (under the Release
+Engineering subproject of SIG Release) which is tasked with publishing
+Kubernetes release artifacts, as well as maintaining the processes and tooling
+to do so.
+
+- [Expectations](#expectations)
+- [Identification](#identification)
+- [Selection](#selection)
+- [Promotion](#promotion)
+
+## Expectations
+
+(From ["Becoming a Release Manager Associate"][becoming-a-release-manager-associate])
+
+Contributors can become Associates by demonstrating the following:
+
+- consistent participation, including 6-12 months of active release
+  engineering-related work
+- experience fulfilling a technical lead role on the Release Team during a
+  release cycle
+  - this experience provides a solid baseline for understanding how SIG Release
+    works overall—including our expectations regarding technical skills,
+    communications/responsiveness, and reliability
+- working on SIG Release items that improve our interactions with Testgrid,
+  cleaning up libraries, etc.
+  - these efforts require interacting and pairing with Release Managers and
+    Associates
+
+There is currently no one accepted process for identifying or selecting
+candidate Release Manager Associates.
+
+## Identification
+
+In the past, we have done the following to consider candidates:
+
+- A candidate expresses interest to a Release Manager or SIG lead
+- A Release Manager or SIG lead recognizes the work of the candidate and
+  suggests they join the team
+- A SIG lead reaches out to current Release Team towards the end of the
+  release to seed candidates
+
+## Selection
+
+Once a candidate has been identified, SIG leads or senior Release Managers will
+do the following:
+
+- discuss the expectations of the role with them
+- confirm interest with the candidate
+- discuss the candidate with other Release Managers and SIG leads
+- vote on confirming the candidate as a Release Manager Associate
+
+If confirmed, members of Release Managers group will work with the candidate
+will work with the new Release Manager Associate on [onboarding][onboarding].
+
+## Promotion
+
+(From ["Becoming a Release Manager"][becoming-a-release-manager])
```

Comment:
OK, my $0.02 here is that we need to funnel the different stepping stones to the RM ladder and make them converge so that candidates start by shadowing a Branch Manager during one release. 

As we've been discussing, we want to open up the possibility of becoming a branch manager to more contributors, not just Release Team leads. This means that candidates will come from other realms and not just the RT but it also means that prior work we can consider as experience as release management experience may vary.

We can consider prior experience from many realms, but a tour as branch manager shadow should be required and should be what ultimately determines fitness for the role. Both giving candidates a real-world view of the RM duties and tooling while always having a guide to show them the way. On the opposite end, the tour through the release cycle will also show SIG Release if that person is willing to have the commitment.

In practice, this means dropping the "_experience fulfilling a technical lead role on the Release Team_" requirement in the [Becoming a Release Manager Associate](https://kubernetes.io/releases/release-managers/#becoming-a-release-manager-associate) section of the doc and coding the requirements to become an RMA as a first set of "soft", subjective requirements and then a "hard" one:

* "Soft": More subjective, consistent work, improving interactions with testgrid, etc
* "Hard": A cycle as Branch Manager shadow.

My opinion is that once both are met, collectively as a group, we can recognize the candidate and accept them as RMA.